lib/omnibus-ctl.rb

```diff
@@ -694,6 +694,11 @@ def run(args)
 
       command_to_run = args[0]
 
+      if command_to_run == "--help"
+        help
+        Kernel.exit 0
+      end
```

Comment:
Would it make sense to leave a comment here to that effect? I was pretty confused here, as well, given the other `help` with tests for it exiting `0`.